### PR TITLE
Qrypto install message

### PR DESCRIPTION
**Pull request by dwalintukan**
- add the install qrypto popup and inline msg
- clean up nav bar and add some fixes
- **not finished. this requires the qrypto feature branch to be merged to master to complete.**

image

### CODE REVIEW

File: src/components/InstallQryptoPrompt/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { withStyles, Typography, Button, Popover, Grid } from '@material-ui/core';
+import { inject, observer } from 'mobx-react';
+import { FormattedMessage } from 'react-intl';
+import cx from 'classnames';
+
+import styles from './styles';
+
+
```
Comment (soshochang) [inline src/components/InstallQryptoPrompt/index.js]: nit: extra space, is it intentional to have 2 lines after the last import?

File: src/components/InstallQryptoPrompt/store.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { action, observable } from 'mobx';
+
+import { urls } from '../../config/app';
+
+
```
Comment (soshochang) [inline src/components/InstallQryptoPrompt/store.js]: nit: space

File: src/components/NavBar/BotCourt.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { withStyles, Button } from '@material-ui/core';
+import { inject, observer } from 'mobx-react';
+import { FormattedMessage } from 'react-intl';
+import { Routes, EventStatus } from 'constants';
+import cx from 'classnames';
+
+import styles from './styles';
+import NavLink from './components/NavLink';
+
+
```
Comment (soshochang) [inline src/components/NavBar/BotCourt.js]: nit: space

File: src/components/NavBar/MyActivities.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import { inject, observer } from 'mobx-react';
+import { FormattedMessage } from 'react-intl';
+import { Routes } from 'constants';
+import cx from 'classnames';
+import { includes } from 'lodash';
+
+import styles from './styles';
+import NavLink from './components/NavLink';
+
+
```
Comment (soshochang) [inline src/components/NavBar/MyActivities.js]: nit:space

**Comment by BernardHan** [comment]
Jest testing failed at UiStore.test.js, this is a snippet used to test if Jest works, you can delete the file to pass the integration, we will write the real test later.

File: src/components/InstallQryptoPrompt/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/components/InstallQryptoPrompt/index.js]: Alex told me before the double line after the imports was standard, but not sure if that was his code style or if its React standard. I prefer 1 line too, will remove all.

**Comment by dwalintukan** [comment]
@BernardHan sounds good. will delete the Jest init test.

**Pull request merged**